### ISSUE TITLE
Default `overrideNative` to `!videojs.browser.IS_SAFARI`

### ISSUE THREAD

**Issue by marcelbeumer** [videojs/http-streaming#505]
## Description

Similar to #168, but instead of defaulting to `true`, I think `overrideNative` should default to `!videojs.browser.IS_SAFARI`, like done here: https://github.com/videojs/http-streaming/blob/57a38e994836b3a4a14c5885b0e7dd444c718238/index.html#L65

It would allow for a more consistent player experience out of the box. I found out about the overrideNative option after finding out Android/Chrome has issues not triggering "waiting" in videojs, first thinking I stumbled on a major vjs bug.

### videojs-http-streaming version
what version of videojs-http-streaming does this occur with?
videojs-http-streaming 1.9.3

### videojs version
what version of videojs does this occur with?
video.js 7.5.4

**Comment by wkoncki** [comment]
@marcelbeumer 
Just wanted to add the default value should probably be
`overrideNative: !videojs.browser.IS_ANY_SAFARI`
since iOS/iPadOS devices have good HLS support just like macOS ones. Using IS_SAFARI would make the exception only cover desktop.

**Comment by gkatsev** [comment]
yeah, we're likely going to use `IS_ANY_SAFARI` rather than `IS_SAFARI` as it covers more bases.

**Comment by blacktrash** [comment]
> yeah, we're likely going to use `IS_ANY_SAFARI` rather than `IS_SAFARI` as it covers more bases.

@gkatsev - so e.g. Chrome on iOS does not have issues with vhs? So far I used `!IS_ANY_SAFARI && !IS_IOS` ...
